.gitignore

```diff
@@ -9,6 +9,9 @@ elm.js
 documentation.json
 docs.json
 
+# elm-verify-examples generated tests
+tests/Doc/OpenSolid
```

Comment:
Very minor point but couldn't this just be tests/Doc? I don't anticipate changing the OpenSolid name but tests/Doc seems a little more direct...just 'ignore the doc tests'!

Reply:
The reason I didn't to tests/Doc is that there is tests/Doc/DocTestStatic. So we would need to either get rid of that or move it up into tests

Reply:
Ah, I see, there's some non-generated code inside the Doc directory

Reply:
Oops, didn't refresh the page (so didn't see your reply) until I posted mine, sorry =)